$NetBSD: patch-wscript,v 1.5 2024/03/18 15:36:15 ryoon Exp $

- Use internal copy of qm-dsp. From Arch Linux.
- Install config files to examples, per pkgsrc policy

--- wscript.orig	2023-12-13 14:20:57.000000000 +0000
+++ wscript
@@ -1140,8 +1140,6 @@ def configure(conf):
 
     if Options.options.use_external_libs:
         conf.define('USE_EXTERNAL_LIBS', 1)
-        conf.env.append_value(
-            'CXXFLAGS', '-I' + Options.options.qm_dsp_include)
 
     if Options.options.boost_include != '':
         conf.env.append_value('CXXFLAGS', '-I' + Options.options.boost_include)
@@ -1645,7 +1643,7 @@ def build(bld):
     if bld.is_defined ('BEATBOX'):
         bld.recurse('tools/bb')
 
-    bld.install_files (bld.env['CONFDIR'], 'system_config')
+    bld.install_files (os.environ['EGDIR'], 'system_config')
 
     bld.install_files (os.path.join (bld.env['DATADIR'], 'templates'), bld.path.ant_glob ('share/templates/**'), cwd=bld.path.find_dir ('share/templates'), relative_trick=True)
     bld.install_files (os.path.join (bld.env['DATADIR'], 'rdf'), bld.path.ant_glob ('share/rdf/*.n3'))
